### PR TITLE
Meteostick: Add example things definition and improve config handling

### PR DESCRIPTION
**Pull request by cdjackson**
Changed mode config to integer from text and actually use it in the initialisation. Added example thing definition.

Signed-off-by: Chris Jackson <chris@cd-jackson.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.meteostick/README.md [hunk 1]
```diff
@@ -19,6 +19,15 @@ The Meteostick things need to be manually added - there is no discovery in the M
 First add and configure the Meteostick bridge - the port and frequency band for your region need to be set.
 Next add the sensor and configure the channel number.
 
+### Thing Configuration
+
+Things can be defined in the .thing file as follows
+
+```
+meteostick:meteostick_bridge:receiver [ port="/dev/tty.usbserial-AI02XA60" mode="1" ]
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.meteostick/README.md]: This part should rather go into the "Full Example" section: https://github.com/eclipse/smarthome/blob/master/tools/archetype/binding/src/main/resources/archetype-resources/README.md#full-example

File: addons/binding/org.openhab.binding.meteostick/README.md [hunk 2]
```diff
@@ -19,6 +19,15 @@ The Meteostick things need to be manually added - there is no discovery in the M
 First add and configure the Meteostick bridge - the port and frequency band for your region need to be set.
 Next add the sensor and configure the channel number.
 
+### Thing Configuration
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.meteostick/README.md]: "This should be mainly about its mandatory and optional configuration parameters"
-> provide a short table that describes the port and mode config parameters.

**Comment by cdjackson** [comment]
Updated...

**Comment by kaikreuzer** [comment]
Wonderful, thanks :-)

**Pull request merged**